front/app/containers/Admin/projects/components/ProjectRow/index.tsx

```diff
@@ -0,0 +1,161 @@
+import React, { useState } from 'react';
+import { isNilOrError } from 'utils/helperUtils';
+import styled from 'styled-components';
+import { adminProjectsProjectPath } from 'containers/Admin/projects/routes';
+
+// i18n
+import { FormattedMessage } from 'utils/cl-intl';
+import messages from '../../messages';
+
+// components
+import {
+  RowContent,
+  RowContentInner,
+  RowTitle,
+  RowButton,
+  ActionsRowContainer,
+} from '../StyledComponents';
+import DeleteProjectButton from '../DeleteProjectButton';
+import PublicationStatusLabel from '../PublicationStatusLabel';
+import { IconNames, StatusLabel } from '@citizenlab/cl2-component-library';
+import Error from 'components/UI/Error';
+import GroupsTag from './GroupsTag';
+import AdminTag from './AdminTag';
+
+// resources
+import { canModerateProject } from 'services/permissions/rules/projectPermissions';
+import useAuthUser from 'hooks/useAuthUser';
+
+// types
+import { IAdminPublicationContent } from 'hooks/useAdminPublications';
+
+export const StyledStatusLabel = styled(StatusLabel)`
+  margin-right: 5px;
+  margin-top: 4px;
+  margin-bottom: 4px;
+`;
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+`;
+
+type CustomButtonAction = {
+  buttonContent: JSX.Element;
+  handler: (publicationId: string) => () => void;
+  icon: IconNames;
+  processing?: boolean;
+};
+type ButtenAction = CustomButtonAction | 'manage' | 'delete';
+
+interface Props {
+  publication: IAdminPublicationContent;
+  actions?: ButtenAction[];
+  hidePublicationStatusLabel?: boolean;
+  className?: string;
+}
+
+const ProjectRow = ({
+  publication,
+  actions,
+  hidePublicationStatusLabel,
+  className,
+}: Props) => {
+  const [isBeingDeleted, setIsBeingDeleted] = useState<boolean>(false);
+  const [deletionError, setDeletionError] = useState<string>('');
+  const authUser = useAuthUser();
+  const projectId = publication.publicationId;
+  const publicationStatus = publication.attributes.publication_status;
+  const userCanModerateProject =
```

Comment:
We should use our permission system for this - see `usePermission` hook.

Reply:
Okay if I not do this now? It's an existing component. usePermission needs some better documentation and types, and I would have to test, etc. 

I'd like to not increase the scope of this PR.

Reply:
Yes, alright. Don't really want to expand work on this but we should really use our permission policies more consistently. Perhaps we can keep it in mind for the future.